lib/scsb/requests.js

```diff
@@ -15,7 +15,7 @@ const _createRecapCodeMap = async (bib) => {
     const client = await scsb.client()
     const __start = new Date()
     // Bib service returns bibId without check digit, but SCSB requires it, as well as .b prefix
-    const scsbFriendlyBibId = '.b' + bNumberWithCheckDigit(bib.id)
+    const scsbFriendlyBibId = '.b' + addSierraCheckDigit(bib.id)
```

Comment:
is this going to add an extra b after sierraCheckDigit adds the prefix back on?

Reply:
No. `addSierraCheckDigit` doesn't add any prefix to that wasn't already there.

Reply:
if addSierraCheckDigit returns b1234x, then scsbFriendlyBibId will be .bb1234x. is that the desired behavior?

Reply:
`addSierraCheckDigit` will only return "b1234x" if `bib.id` is "b1234". But `bib.id` is "1234", so `addSierraCheckDigit('1234') => '1234x'`